src/componentes/CriarSenha/CriarSenha.stories.js

```
@@ -1,76 +1,69 @@
+		method: "post",
+		url: "http://localhost:8000/suporte/usuarios/cadastro",
+		headers: {
+			...data.getHeaders(),
+		},
+		data: data,
+	};
+
+	return await axios(config)
+		.then((response) => response.data)
+		.catch((error) => {
+			console.log(error);
```

Biome (1.9.4)

[error] Don't use console.log

console.log is usually a tool for debugging and you don't want to have that in production.
If it is not for debugging purpose then using console.info might be more appropriate.
Unsafe fix: Remove console.log


(lint/suspicious/noConsoleLog)